{
    "CVE_data_meta": {
        "ASSIGNER": "cve@mitre.org",
        "ID": "CVE-2020-8631",
        "STATE": "PUBLIC"
    },
    "affects": {
        "vendor": {
            "vendor_data": [
                {
                    "product": {
                        "product_data": [
                            {
                                "product_name": "n/a",
                                "version": {
                                    "version_data": [
                                        {
                                            "version_value": "n/a"
                                        }
                                    ]
                                }
                            }
                        ]
                    },
                    "vendor_name": "n/a"
                }
            ]
        }
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
        "description_data": [
            {
                "lang": "eng",
                "value": "cloud-init through 19.4 relies on Mersenne Twister for a random password, which makes it easier for attackers to predict passwords, because rand_str in cloudinit/util.py calls the random.choice function."
            }
        ]
    },
    "problemtype": {
        "problemtype_data": [
            {
                "description": [
                    {
                        "lang": "eng",
                        "value": "n/a"
                    }
                ]
            }
        ]
    },
    "references": {
        "reference_data": [
            {
                "url": "https://github.com/canonical/cloud-init/pull/204",
                "refsource": "MISC",
                "name": "https://github.com/canonical/cloud-init/pull/204"
            },
            {
                "url": "https://bugs.launchpad.net/ubuntu/+source/cloud-init/+bug/1860795",
                "refsource": "MISC",
                "name": "https://bugs.launchpad.net/ubuntu/+source/cloud-init/+bug/1860795"
            },
            {
                "refsource": "MLIST",
                "name": "[debian-lts-announce] 20200221 [SECURITY] [DLA 2113-1] cloud-init security update",
                "url": "https://lists.debian.org/debian-lts-announce/2020/02/msg00021.html"
            },
            {
                "refsource": "SUSE",
                "name": "openSUSE-SU-2020:0400",
                "url": "http://lists.opensuse.org/opensuse-security-announce/2020-03/msg00042.html"
            }
        ]
    }
}